extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -36,8 +39,123 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
+  text-overflow: ellipsis !important;
+  overflow: hidden !important;
+  margin-bottom: 0.15em !important;
+}
+
+.i-amphtml-story-subscriptions-subtitle-first, .i-amphtml-story-subscriptions-subtitle-second {
+  font-size: 0.9em !important;
+  letter-spacing: -0.015em !important;
+  text-overflow: ellipsis !important;
+  overflow: hidden !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-button, .i-amphtml-story-subscriptions-google-button {
+  margin: 0 auto 0.8em !important;
+  padding: 0 !important;
+  border: none !important;
+  width: 12.2em !important;
+  height: 2.3em !important;
+  border-radius: 51px !important;
+  display: flex !important;
+  align-items: center !important;
+  cursor: pointer !important;
+}
+.i-amphtml-story-subscriptions-publisher-button {
+  margin-top: 2.48em !important;
+  background: black !important;
+}
+.i-amphtml-story-subscriptions-google-button {
+  background: #F2F2F2 !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-logo, .i-amphtml-story-subscriptions-google-logo {
+  width: 1.85em !important;
+  height: 1.85em !important;
+  border-radius: 50% !important;
+  margin-inline: 0.25em 0.65em !important;
+}
+.i-amphtml-story-subscriptions-google-logo {
+  background-size: contain !important;
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="32px" height="32px" fill="none"><path fill="#fff" d="M16 32a16 16 0 1 0 0-32 16 16 0 0 0 0 32Z"/><path fill="#4285F4" d="M28.56 13.78H16v5h7.19a6.19 6.19 0 0 1-2.67 3.99v3.33h4.3c3.34-3.1 4.46-7.93 3.74-12.32Z"/><path fill="#34A853" d="M16 29.33c3.6 0 6.61-1.2 8.81-3.23l-4.29-3.33A8.04 8.04 0 0 1 16 24.06a7.96 7.96 0 0 1-7.48-5.52H4.1v3.44A13.31 13.31 0 0 0 16 29.33Z"/><path fill="#FBBC05" d="M8.52 18.54a7.93 7.93 0 0 1 0-5.08v-3.44H4.1a13.18 13.18 0 0 0 0 11.96l4.42-3.44Z"/><path fill="#EA4335" d="M16 7.94c1.97 0 3.72.68 5.11 2l3.8-3.8A12.75 12.75 0 0 0 16 2.67c-5.21 0-9.71 3-11.9 7.35l4.42 3.44A7.96 7.96 0 0 1 16 7.94Z"/></svg>') !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-button-text, .i-amphtml-story-subscriptions-google-button-text {
+  letter-spacing: 0.01em !important;
+  font-size: 0.75em !important;
+  font-weight: 400 !important;
+  font-family: Google Sans, sans-serif !important;
+  max-width: 12.6em !important;
+  text-align: initial !important;
+  white-space: normal !important;
+
+  /* line clamping and ellipses */
+  display: -webkit-box !important;
+  -webkit-line-clamp: 2 !important;
+  -webkit-box-orient: vertical !important;
+  overflow: hidden !important;
+}
+.i-amphtml-story-subscriptions-publisher-button-text {
+  color: white !important;
+}
+.i-amphtml-story-subscriptions-google-button-text {
+  color: #5E5E5E !important;
+}
+
+.i-amphtml-story-subscriptions-signin {
+  margin-top: 2.75em !important;
+  margin-bottom: 1.4em !important;
+  font-size: 0.7em !important;
+}
+
+.i-amphtml-story-subscriptions-signin a {
+  font-weight: bold !important;
```

Comment:
This is heavier than what's in the mocks. 
Please use the specific weight
<img width="250" alt="Screen Shot 2022-04-13 at 3 26 31 PM" src="https://user-images.githubusercontent.com/3860311/163264612-420229a8-7042-4a68-8c5b-1b1e432f7baa.png">
.